pybloqs/htmlconv/html_converter.py

```diff
@@ -53,6 +53,15 @@ def write_html_to_tempfile(block, content):
             f.write(content)
         return html_filename
 
+    @staticmethod
+    def remove_temporary_files(temp_files):
+        try:
+            if user_config.get('remove_temp_files', True):
+                for f in temp_files:
+                    os.remove(f)
+        except (IOError, OSError):
```

Comment:
Why expect an IOError or OSError? If there is no local config file, we should always have the default defined in config.py.

Reply:
The exception is for the filesystem operation (obviously ;) ).
Could you please wrap it inside the if statement?
It would be very useful to have the logger output which file could not be removed.

Reply:
Done. Thanks for the suggestion.